forwarder/internal/logs/parse.go

```diff
@@ -142,6 +142,13 @@ func (f FunctionAppParser) Parse(scanner *bufio.Scanner, blob storage.Blob, piiS
 				yield(response)
 				return
 			}
+
+			// Note that function app logs do not have parsable resource IDs that we can get a log source from.
+			if len(currLog.Source) == 0 {
+				currLog.Source = functionAppSource
+				currLog.Tags = append(currLog.Tags, "source:"+functionAppSource)
```

Comment:
This happens by default when the Source is set.
https://github.com/DataDog/azure-log-forwarding-orchestration/blob/main/forwarder/internal/logs/datadog.go#L109

Reply:
Ah good catch. I'm not sure why they are doubly set, but worth maintaining the behavior for now